src/components/panel/panel.js

```diff
@@ -661,6 +676,9 @@ MdPanelRef.prototype.attach = function() {
   var self = this;
   this._attachPromise = this._$q(function(resolve, reject) {
     self._createPanel().then(function() {
+      self._$rootScope.$applyAsync(function() {
```

Comment:
I am using md-autofocus and it hasn't added the class _md-autofocus by this point. Without that class, the mdUtil.findFocusTarget can't find the right spot. I don't know if we need to fix something in createPanel to wait longer, but this is the fix I found. Do you have a different suggestion for how to wait longer to determine what to focus?